Plain text feed (#15254)
demisto/content: Packs/FeedPlainText/ReleaseNotes/1_1_6.md

#### Integrations
##### Plain Text Feed

> - 'custom' type was added to indicator_type single select param.
> - If selected, user will have to insert custom indicator type at the according field.
> - If selected but no value was inserted at 'custom_indicator_type', error will be raised.

Reviewer: ```suggestion
- Added an option to select a custom indicator type in the *Indicator Type* parameter. If selected, the *Custom Indicator Type* parameter must be provided.
```
